tests/integration/helpers.bash

```diff
@@ -18,7 +18,7 @@ HELLO_IMAGE="$TESTDATA/hello-world.tar"
 HELLO_BUNDLE="$BATS_TMPDIR/hello-world"
 
 # CRIU PATH
-CRIU="$(which criu)"
+CRIU="$(which criu)" || true
```

Comment:
How does this fix the problem? As far as I can tell, you're setting the value of `CRIU` to be ` || true`. To be fair, that path probably doesn't exist, but it doesn't mean the logic makes sense.